.env

```diff
@@ -0,0 +1,4 @@
+export FLASK_APP=run.py
+export FLASK_ENV=development
+export DB_TEST="dbname='send_it_test' host='localhost' port='5432' user='postgres' password='v0k3ch!97'"
+export DB_PRODUCTION="dbname='send_it' host='localhost' port='5432' user='postgres' password='v0k3ch!97'"
```

Comment:
You never want to store sensitive information, like your database password, in your source code. If your source code is ever seen by anyone, they now have access to your database.
You should conceal/hide your production password in an environment variable.

Also create a `DB_DEVELOPMENT` database, best without username and password (I prefer you only set up a password for your production db)